src/api/certificates.tsx

```diff
@@ -1,2 +1,20 @@
+import { handleResponse } from "util/helpers";
+
 export const checkAuth = () =>
   fetch("/1.0/certificates").then((response) => response.status !== 403);
+
+export const addCertificate = (token: string) => {
+  return new Promise((resolve, reject) => {
+    fetch(`/1.0/certificates`, {
+      method: "POST",
+      body: JSON.stringify({
+        type: "client",
+        password: token,
+        name: "some-name",
```

Comment:
Is the name mandatory? What's its purpose? (where would it be displayed/stored?)
I wonder if we should set some more appropriate value here... :sweat_smile:

Reply:
I think we don't need it - let me test it. If I remember correctly, the name when creating the token via cli is used.